src/main/resources/schema/ans/0.5.9/utils/reference.json

```diff
@@ -44,7 +44,8 @@
         },
         "provider": {
           "description": "A URL that can resolve the id into an ANS element",
-          "type": "string"
+          "type": "string",
+          "format": "uri"
```

Comment:
Ok.  This serves as a pretty good argument for getting fields like this validated from the start.

This alternative, while not the prettiest thing in the world, passes all tests and would allow the old content to be grandfathered.

There was also talk about deprecating/removing this field altogether—should that just happen for 0.5.9, and then we can leave this unvalidated?